homeassistant/components/aladdin_connect/__init__.py

```diff
@@ -1,28 +1,36 @@
 """The aladdin_connect component."""
+import asyncio
+from datetime import timedelta
 import logging
 from typing import Final
 
-from aladdin_connect import AladdinConnectClient
+from AIOAladdinConnect import AladdinConnectClient
+from aiohttp import ClientConnectionError
 
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, Platform
 from homeassistant.core import HomeAssistant
-from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
 
 from .const import DOMAIN
 
 _LOGGER: Final = logging.getLogger(__name__)
 
 PLATFORMS: list[Platform] = [Platform.COVER]
+SCAN_INTERVAL = timedelta(seconds=120)
```

Comment:
This constant has no effect in this module. It only has an implicit effect in platform modules.